source/includes/properties/_05-create.md.erb

```diff
@@ -0,0 +1,32 @@
+## Create property
+
+```shell
+$ curl 'http://ci.example.com/go/properties/PipelineName/541/StageName/1/JobName/PropertyName' \
+        -u 'username:password' \
+        -X POST \
+        -d 'value=PropertyValue'
+```
+
+> The above command defines a property ‘PropertyName’ for ‘PipelineName/541/StageName/1/JobName’ and sends the response this:
```

Comment:
Also it appears that this text was copy pasted from the HTML, as a result it contains curly quotes (‘) as against neutral quotation marks(').